### DISCUSSION TITLE
whitch version of  ifcconvert download link in http://ifcopenshell.org/ifcconvert?

### DISCUSSION THREAD

**Discussion by hxboxy-xh** [IfcOpenShell/IfcOpenShell#1712] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/32007375/131433893-3b13e6a6-8651-4778-be67-3920753bb7c9.png)
the 0.6.0 latest  ifcconvert  
whitch tag of the sourcecode?

**Comment by aothms** [comment, marked as answer]
https://github.com/IfcOpenShell/IfcOpenShell/issues/1713
